src/dirdef.h

```diff
@@ -34,7 +36,7 @@ class FTextStream;
 class DirDef;
 
 /** A list of directories. */
-class DirList : public QList<DirDef>
+class DirList : public std::deque<DirDef>
```

Comment:
I've learned that it is not a good idea to inherit from STL containers, since they do not have virtual destructors. See also: https://stackoverflow.com/questions/2034916/is-it-okay-to-inherit-implementation-from-stl-containers-rather-than-delegate

Reply:
Thank you for pointing that out. In the [article](https://punchlet.wordpress.com/2009/12/29/letter-the-fifth/) [referenced](https://stackoverflow.com/questions/2034916/is-it-okay-to-inherit-implementation-from-stl-containers-rather-than-delegate#comment1959577_2034916) by the resource you gave, it is well explained why one should not inherit from the STL containers. Thus I now applied an alternative discussed in that article: Free functions.